### PR TITLE
520changelog

### PR DESCRIPTION
**Pull request by v-kaywon**
This change is [Reviewable](https://reviewable.io/reviews/microsoft/msphpsql/722)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,61 @@ All notable changes to this project will be documented in this file.
...
+## Windows/Linux/macOS 5.2.0 - 2017-02-14
+Updated PECL release packages. Here is the list of updates:
+
+### Added
+- Added support for Always Encrypted with basic CRUD functionalities (see [here](https://github.com/Microsoft/msphpsql/wiki/Features#aebindparam))
+  - Support for Windows Certificate Store (use connection keyword ColumnEncryption)
+  - Support for inserting into and modifying an encrypted column
+  - Support for fetching from an encrypted column
+- Added support for PHP 7.2
+- Added support for MSODBC 17
```
Comment (lilgreenbird) [inline CHANGELOG.md]: can we be a bit more consistent with the naming? Sometimes we say "MSODBC" and sometimes "ODBC Driver". I think it should be "MS ODBC Driver", what do you think?

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,61 @@ All notable changes to this project will be documented in this file.
...
+- Added support for Ubuntu 17 (requires MSODBC 17)
+- Added support for Debian 9 (requires MSODBC 17)
+- Added support for SUSE 12
+- Added Driver option to set the ODBC driver, Added "Driver" option, valid values are ODBC Driver 17 for SQL Server,ODBC Driver 13 for SQL Server, and ODBC Driver 11 for SQL Server
```
Comment (lilgreenbird) [inline CHANGELOG.md]: minor pickiness, space after ","

Comment (lilgreenbird) [reply CHANGELOG.md]: also, quotes around "ODBC Driver xx"?

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,61 @@ All notable changes to this project will be documented in this file.
...
+- Support for non-UTF8 locales in Linux and macOS
+- Fixed crash caused by executing an invalid query in a transaction (Issue [#434](https://github.com/Microsoft/msphpsql/issues/434))
+- Added error handling for using PDO::SQLSRV_ATTR_DIRECT_QUERY or PDO::ATTR_EMULATE_PREPARES in a Column Encryption enabled connection
+- Added error handling for binding TEXT, NTEXT or IMAGE as output parameter (Issue [#231](https://github.com/Microsoft/msphpsql/issues/231))
+- PDO::quote with string containing ASCII NUL character (Issue [#538]( https://github.com/Microsoft/msphpsql/issues/538))
+- Decimal types with no decimals are correctly handled when AE is enabled (PR [#544](https://github.com/Microsoft/msphpsql/pull/544))
+- BIGINT as an output param no longer results in value out of range exception when the returned value is larger than a maximum integer ([PR #567](https://github.com/Microsoft/msphpsql/pull/567))
+
+### Removed
+- No longer support Ubuntu 15
```
Comment (david-puglielli) [inline CHANGELOG.md]: I think it's better to say "Dropped support for Ubuntu 15"

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,61 @@ All notable changes to this project will be documented in this file.
...
+- Supplying tablename into PDO::lastInsertId($name) no longer return the last inserted row ([lastInsertId](https://github.com/Microsoft/msphpsql/wiki/Features#lastinsertid))
```
Comment (david-puglielli) [inline CHANGELOG.md]: This is mentioned in the fixed section, no need to mention it here.

File: CHANGELOG.md [hunk 5]
```diff
@@ -3,6 +3,61 @@ All notable changes to this project will be documented in this file.
...
+
+### Limitations
+- Always Encrypted is not support in Linux and macOS
```
Comment (david-puglielli) [inline CHANGELOG.md]: *supported

File: CHANGELOG.md [hunk 6]
```diff
@@ -3,6 +3,61 @@ All notable changes to this project will be documented in this file.
...
+- In Linux and macOS, setlocale() only takes effect if it is invoked before the first connection. The subsequent locale setting will not work
```
Comment (david-puglielli) [inline CHANGELOG.md]: 'The subsequent locale setting will not work' - this is awkward. Maybe 'Attempting to set the locale after connection will not work'?

File: CHANGELOG.md [hunk 7]
```diff
@@ -3,6 +3,61 @@ All notable changes to this project will be documented in this file.
...
+- Always Encrypted functionalities are only supported using MSODBC 17
+- [Always Encrypted limitations](https://github.com/Microsoft/msphpsql/wiki/Features#aelimitation)
+- When using sqlsrv_query with Always Encrypted feature, SQL type has to be specified for each input (see [here](https://github.com/Microsoft/msphpsql/wiki/Features#aebindparam))
+- No support for inout / output params when using sql_variant type
+
+### Known Issues
+- Connection pooling on Linux doesn't work properly when using the MSODBC 17
```
Comment (david-puglielli) [inline CHANGELOG.md]: no need for 'the'

File: CHANGELOG.md [hunk 8]
```diff
@@ -3,6 +3,61 @@ All notable changes to this project will be documented in this file.
...
+- When pooling is enabled in Linux or macOS
+  - unixODBC <= 2.3.4 (Linux and macOS) might not return proper diagnostics information, such as error messages, warnings and informative messages
+  - due to this unixODBC bug, fetch large data (such as xml, binary) as streams as a workaround. See the examples [here](https://github.com/Microsoft/msphpsql/wiki/Connection-Pooling-on-Linux-and-Mac)
+- Connection with Connection Resiliency enabled so not resume properly with Connection Pooling (Issue [#678](https://github.com/Microsoft/msphpsql/issues/678))
```
Comment (david-puglielli) [inline CHANGELOG.md]: I think this should be 'Connection with Connection Resiliency enabled *does* not resume properly with Connection Pooling *enabled*'?

File: CHANGELOG.md (hunk 1 above)
Comment (v-kaywon) [reply CHANGELOG.md]: sounds good

File: CHANGELOG.md (hunk 4 above)
Comment (v-kaywon) [reply CHANGELOG.md]: I think Jenny agreed to add it in both places since it is a breaking change. I'll leave it here.

**Pull request merged**